README.md

```diff
@@ -163,6 +163,15 @@ ssh and set root password using the following kernel command line options:
 fdi.ssh=1 fdi.rootpw=redhat
 ```
 
+Root password can also be specified in encrypted form (using 'redhat' as
+an example below). Single and/or double quotes around password are 
+recommended to be used to prevent possible special characters
+interpretation.
+
+```
+fdi.rootpw='$1$_redhat_$i3.3Eg7ko/Peu/7Q/1.wJ/'
+```
```

Comment:
I still don't like the quotes, kernel command line does not have shell-like parsing or interpolation. More than that, passwd library does use base64 encoding which is safe.